### ISSUE TITLE
Browser (Desktop) crashes regardless of activity. 

### ISSUE THREAD

**Issue by Sarafian** [brave/brave-browser#10107]
## Description 
Brave desktop browser starts fine and works fine but suddenly after some time the browser crashes

## Steps to Reproduce

   1.  Open Brave (Desktop)
   2. Browse around or just do something else in a different app
   3.  Process crashes/exits

## Actual result:

You can't work with the browser.

## Expected result:

Normal work

## Reproduces how often: 

Every time even if you open in private mode or tor tabs

## Brave version (brave://version info)

Brave | 1.9.76 Chromium: 81.0.4044.138 (Official Build) (64-bit)
-- | --
Revision | 8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{#999}
OS | Windows 10 OS Version 2004 (Build 19041.264)
JavaScript | V8 8.1.307.32
Flash | (Disabled)
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
Command Line | "C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --sync-url=https://no-thanks.invalid --enable-features=MixedContentSiteSetting,SimplifyHttpsIndicator,PassiveMixedContentWarning,AutoupgradeMixedContent,WebUIDarkMode,PasswordImport,DnsOverHttps --disable-features=TextFragmentAnchor,AutofillServerCommunication,AudioServiceOutOfProcess,SmsReceiver,VideoPlaybackQuality,LookalikeUrlNavigationSuggestionsUI,AllowPopupsDuringPageUnload,NotificationTriggers --flag-switches-begin --enable-features=MixedContentSiteSetting,SimplifyHttpsIndicator,PassiveMixedContentWarning,AutoupgradeMixedContent,WebUIDarkMode,PasswordImport,DnsOverHttps,BraveSync --flag-switches-end --enable-audio-service-sandbox --disable-webrtc-apm-in-audio-service
Executable Path | C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe
Profile Path | C:\Users\username\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
Variations | 5e3a236d-4113a79e

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes. 1.9.76 is the latest one. Actually this started on Monday 1/6/2020 when I believe I got upgraded to 1.9.76
- Can you reproduce this issue with the beta channel? I'm not with beta
- Can you reproduce this issue with the dev channel? I'm not with dev
- Can you reproduce this issue with the nightly channel? Haven't tried

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? Didn't try. When browser is idle it still crashes
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No. I use Chrome now for daily driver

## Miscellaneous Information:

What I've noticed is that the time required to crash is always the same. Feels like some sort of timeout that leads to a process crashing

**Comment by bsclifton** [comment]
@Sarafian do you have sync enabled? After launching, please visit brave://flags/#brave-sync and disable this setting

If so - sync v1 is going away in our next major release - 1.12.x in favor of a completely new fully featured system (basically, what you get with Chrome/Chromium by default). With the next release, you'll have to create a new sync group - but you'll also have access to all the sync types (autofill data, bookmarks, passwords, etc)

**Comment by Sarafian** [comment]
@bsclifton initially I had tried this because i knew the sync feature was not stable and hence you needed to enable it from the flags area.

But it didn't make a difference and I thought this was not the case. For a couple of weeks I had switched back to Chrome. Then after some releases of Brave, I switched back and it was working. Then I experimented with the flags turning sync on and off and this where I confirm that the sync is the issue. 

So I've been waiting patiently for the synchronization feature and it is good that you mention that is going to be available with the new version.

Thank you.
